internal/cryptocore/nonce.go

```diff
@@ -16,6 +18,18 @@ func RandBytes(n int) []byte {
 	return b
 }
 
+// RandBytesDevRandom gets "n" random bytes from /dev/random or panics
+func RandBytesDevRandom(n int) []byte {
+	f, err := os.Open("/dev/random")
+	defer f.Close()
+	b := make([]byte, n)
+	_, err = io.ReadFull(f, b)
+	if err != nil {
+		log.Panic("Failed to read random bytes: " + err.Error())
+	}
+	return b
+}
+
```

Comment:
Because this function should not be used anywhere but in CreateConfFile, I think it should move to config_file.go and be unexported (lowercase first letter).